src/sentry/integrations/vsts/webhooks.py

```diff
@@ -120,11 +125,24 @@ def handle_assign_to(self, integration, external_issue_key, assigned_to):
             return
         new_value = assigned_to.get('newValue')
         if new_value is not None:
-            email = self.parse_email(new_value)
+            try:
+                email = self.parse_email(new_value)
+            except AttributeError as e:
+                logger.info(
+                    'vsts.failed-to-parse-email-in-handle-assign-to',
+                    extra={
+                        'error': six.text_type(e),
+                        'integration_id': integration.id,
+                        'assigned_to_values': assigned_to,
+                        'external_issue_key': external_issue_key,
+                    }
+                )
+                return  # TODO(lb): return if cannot parse email?
             assign = True
         else:
             email = None
             assign = False
+        # TODO(lb): should there be a logging statement here? or a try-catch?
```

Comment:
Does this work interact with external systems? If no then exceptions could be valid problems with sentry and not something we would want to catch and log.